.babelrc

```diff
@@ -1,7 +1,18 @@
 {
-  "presets": ["@babel/preset-env", "@babel/preset-typescript"],
+  "presets": [
+    [
+      "@babel/preset-env",
+      {
+        "loose": true,
+        "modules": false
+      }
+    ],
+    "@babel/preset-typescript"
+  ],
   "plugins": [
-    "@babel/plugin-proposal-class-properties",
+    ["@babel/plugin-proposal-class-properties", { "loose": false }],
+    ["@babel/plugin-proposal-private-methods", { "loose": false }],
+    ["@babel/plugin-proposal-private-property-in-object", { "loose": false }],
```

Comment:
I don't think we need these. Especially as we use typescript which has its own syntax for private fields.

Reply:
it errors otherwise with the loose transform but I'll actually turn that off